src/components/navbar/navbar.tsx

```diff
@@ -53,6 +53,10 @@ const ChaiseNavbar = (): JSX.Element => {
   const [showRidSpinner, setShowRidSpinner] = useState(false);
   const [topBanners, setTopBanners] = useState<NavbarBanner[]>([]);
   const [bottomBanners, setBottomBanners] = useState<NavbarBanner[]>([]);
+  /**
+   * Keeps track of most recently opened dropdown
+   */
+  const [openDropDown, setOpenDropDown] = useState<number | undefined>(undefined);
```

Comment:
Let's rename this to `openedDropdownIndex`. You can also just use `number` as type and not pass anything for the default value. In the comment you should also mention why you've added it (mention the bug)

```tsx
  const [openedDropdownIndex, setOpenedDropdownIndex] = useState<number>();
```